app/response/utils.py

```diff
@@ -626,15 +646,23 @@ def _closing_email_handler(request_id, data, page, agency_name, email_template):
 
     :return: the HTML of the rendered template of a closing
     """
-    reasons = [Reasons.query.filter_by(id=reason_id).one().content
-               for reason_id in data.getlist('reason_ids[]')]
-    header = "The following will be emailed to the Requester:"
     if eval_request_bool(data['confirmation']):
+        header = "The following will be emailed to the Requester:"
```

Comment:
Maybe for future. I think this should just be a constant string. We use it 18 times so far